artifacts/simple-image/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM fedora
+ADD namespace-reservation-server /usr/bin
```

Comment:
This implies building via Makefile, whereas we aim to build with Docker strategy in cluster. Example of how we do that: https://github.com/openshift/online-archivist/blob/master/Dockerfile

Reply:
> This implies building via Makefile, whereas we aim to build with Docker strategy in cluster. Example of how we do that: https://github.com/openshift/online-archivist/blob/master/Dockerfile

I need one for rapid development.  Feel free to create another for deploying a different way.